docs/LOCALHOST.md

```diff
@@ -22,9 +23,9 @@ Instructions for running a local "developer" environment on localhost:
 7. Run `yarn build`
 8. Setup environment variables for running
    [meadowlark-fastify](../Meadowlark-js/services/meadowlark-fastify/readme.md) service.
-9. In your command prompt navigate to `/Meadowlark-js/services/fastify`
-10. Run `yarn start:local` to start the fastify service
-
+9. Setup environment variables for OAuth. See [OAUTH2](OAUTH2.md) for more details.
+10. In your command prompt navigate to `/Meadowlark-js/services/meadowlark-fastify`
```

Comment:
This step is not necessary since yarn start:local can be run from the Meadowlark-js folder

Reply:
How?

Reply:
I didn't realize this was possible either, but I was able to start fastify from the Meadowlark-js folder